src/main/java/uk/gov/companieshouse/kafka/EmailKafkaProducer.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.kafka;
+
+import java.util.Date;
+import java.util.concurrent.ExecutionException;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.kafka.exceptions.SerializationException;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.producer.ProducerConfig;
+import uk.gov.companieshouse.kafka.producer.CHKafkaProducer;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+import uk.gov.companieshouse.kafka.serialization.SerializerFactory;
+
+@Service
+public class EmailKafkaProducer {
+
+    private final SerializerFactory serializerFactory;
+    private final CHKafkaProducer chKafkaProducer;
+    private static final String EMAIL_SEND_TOPIC = "email-send";
```

Comment:
Static members should be first
```suggestion
    private static final String EMAIL_SEND_TOPIC = "email-send";
    private final SerializerFactory serializerFactory;
    private final CHKafkaProducer chKafkaProducer;
```

Reply:
Logic extracted to new package, suggested change applied there. Marking as resolved.